src/integrate/orchestrator.rs

```diff
@@ -0,0 +1,1204 @@
+                }),
+                None,
+                None,
+            )
+            .try_collect::<Vec<_>>()
+            .await
+            .map_err(|_| "Unable to create image".to_string())?;
+
+        let mut port_bindings = HashMap::new();
+        port_bindings.insert(
+            format!("{}/tcp", devnet_config.bitcoin_controller_port),
+            Some(vec![PortBinding {
+                host_ip: Some(String::from("0.0.0.0")),
+                host_port: Some(format!("{}/tcp", devnet_config.bitcoin_controller_port)),
+            }]),
+        );
+        port_bindings.insert(
+            format!("{}/tcp", devnet_config.bitcoin_node_p2p_port),
+            Some(vec![PortBinding {
+                host_ip: Some(String::from("0.0.0.0")),
+                host_port: Some(format!("{}/tcp", devnet_config.bitcoin_node_p2p_port)),
+            }]),
+        );
+        port_bindings.insert(
+            format!("{}/tcp", devnet_config.bitcoin_node_rpc_port),
+            Some(vec![PortBinding {
+                host_ip: Some(String::from("0.0.0.0")),
+                host_port: Some(format!("{}/tcp", devnet_config.bitcoin_node_rpc_port)),
+            }]),
+        );
+
+        let bitcoind_conf = format!(
+            r#"
+server=1
+regtest=1
+rpcallowip=0.0.0.0/0
+rpcallowip=::/0
+rpcuser={}
+rpcpassword={}
+txindex=1
+listen=1
+reindex=0
+discover=0
+dns=0
+dnsseed=0
+listenonion=0
+rpcserialversion=0
+rpcworkqueue=100
+disablewallet=0
+fallbackfee=0.00001
+
+[regtest]
+bind=0.0.0.0:{}
+rpcbind=0.0.0.0:{}
+rpcport={}
+"#,
+            devnet_config.bitcoin_node_username,
+            devnet_config.bitcoin_node_password,
+            devnet_config.bitcoin_node_p2p_port,
+            devnet_config.bitcoin_node_rpc_port,
+            devnet_config.bitcoin_node_rpc_port,
+        );
+        let mut bitcoind_conf_path = PathBuf::from(&devnet_config.working_dir);
+        bitcoind_conf_path.push("conf/bitcoin.conf");
+        let mut file = File::create(bitcoind_conf_path).expect("Unable to create bitcoind.conf");
+        file.write_all(bitcoind_conf.as_bytes())
+            .expect("Unable to write bitcoind.conf");
+
+        let bitcoin_controller_conf = format!(
+            r#"
+[network]
+rpc_bind = "0.0.0.0:{}"
+block_time = {}
+miner_address = "{}"
+bitcoind_rpc_host = "0.0.0.0:{}"
+bitcoind_rpc_user = "{}"
+bitcoind_rpc_pass = "{}"
+whitelisted_rpc_calls = [
+    "listunspent",
+    "listwallets",
+    "createwallet",
+    "importaddress",
+    "sendrawtransaction",
+    "getrawtransaction",
+    "scantxoutset",
+    "getrawmempool",
+    "getblockhash",
+]
+# Expedite the key registration / genesis block
+[[blocks]]
+count = 1
+block_time = 30000
+ignore_txs = false
+
+# Expedite the key registration / genesis block
+[[blocks]]
+count = 2
+block_time = 20000
+ignore_txs = false
+
+# Give more time to the first blocks
+[[blocks]]
+count = 3
+block_time = 45000
+ignore_txs = false
+"#,
```

Comment:
It might be a bit difficult to find these configs in the source code, would it be possible to separate them out into separate files that get included in the shipped binary?

Reply:
I'll keep this idea in mind for a future release :)